strongbox-storage/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/services/impl/ArtifactSearchServiceImpl.java

```diff
@@ -38,94 +52,166 @@
     @Autowired
     private ConfigurationManager configurationManager;
 
+    @Autowired
```

Comment:
In the master, all occurrences of `@Autowired` are replaced with `@Injected`. Could you please make sure this is the case across your branch as well? Thanks! :)